### PR TITLE
Set ffmpeg process priority

### PR DESCRIPTION
**Pull request by hanabi1224**

### CODE REVIEW

File: Xabe.FFmpeg/FFprobeWrapper.cs [hunk 1]
```diff
@@ -90,29 +91,17 @@ public async Task<string> Start(string args)
...
-                    }
-
-                    return output;
+                    process.WaitForExit();
+                    return process.StandardOutput.ReadToEnd();
                 }
-            });
+            },
+            CancellationToken.None,
+            TaskCreationOptions.LongRunning,
```
Comment (hanabi1224) [inline Xabe.FFmpeg/FFprobeWrapper.cs]: Use dedicate thread instead of thread pool thread here

File: Xabe.FFmpeg.Test/ConversionResultTests.cs [hunk 2]
```diff
@@ -1,23 +1,26 @@
...
 
 namespace Xabe.FFmpeg.Test
 {
     public class ConversionResultTests
     {
-        [Fact]
-        public async Task ConversionResultTest()
+        [Theory]
+        [InlineData(null)]
+        [InlineData(ProcessPriorityClass.High)]
```
Comment (hanabi1224) [inline Xabe.FFmpeg.Test/ConversionResultTests.cs]: (Locally) I got 50% perf gain with high priority

**Comment by tomaszzmuda** [comment]
Nice work 👍 
Great idea, just please make the CI pass and I will merge this.

File: Xabe.FFmpeg/FFmpegWrapper.cs [hunk 3]
```diff
@@ -33,49 +34,30 @@ internal class FFmpegWrapper : FFmpeg
...
+                    using (var processEnded = new ManualResetEvent(false))
                     {
-                        process.WaitForExit();
-
-                        cancellationToken.ThrowIfCancellationRequested();
-                        if (_wasKilled)
-                        {
-                            throw new ConversionException("Cannot stop process. Killed it.", args);
-                        }
+                        processEnded.SafeWaitHandle = new SafeWaitHandle(process.Handle, false);
```
Comment (hanabi1224) [inline Xabe.FFmpeg/FFmpegWrapper.cs]: For some reason this works on windows only, I will revert this change

File: Xabe.FFmpeg/FFmpegWrapper.cs [hunk 4]
```diff
@@ -49,25 +51,49 @@ internal Task<bool> RunProcess(string args, CancellationToken cancellationToken)
...
-                        }
-                        catch (InvalidOperationException)
-                        {
                         }
                     });
 
                     using (ctr)
                     {
-                        process.WaitForExit();
+                        using (var processEnded = new ManualResetEvent(false))
```
Comment (hanabi1224) [inline Xabe.FFmpeg/FFmpegWrapper.cs]: RTSP_NotExistingStream_CancelledSucesfully stucks on windows cli dotnet test run without this change (both locally and appveyor)

**Pull request merged**